crates/bevy_transform/src/components/transform.rs

```diff
@@ -120,11 +171,14 @@ impl Transform {
         value
     }
 
+    /// Change the `scale` of this [`Transform`].
```

Comment:
We should probably be more clear that this multiplies the current scale by the given scale.